xml/art_sle_ha_install_quick.xml

```diff
@@ -788,6 +793,98 @@ softdog                16384  1</screen>
        <keycap function="control"/><keycap>C</keycap>
       </keycombo>.
    </para>
+   </sect2>
+
+   <sect2 xml:id="sec-ha-inst-quick-test-with-cluster-script">
+    <title>Testing with the &test.script; Script</title>
+    <para>
+     The command &cmd.test.script; is a tool to run standardized tests
+     for a cluster.
+     It triggers cluster failures and verify configuration to find
+     problems. Before you use your cluster in production, it is
+     recommended to use this script to make sure everything works as expected.
+    </para>
+    <para>
+     The script supports the following checks:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <formalpara>
+       <title>Environment Check <option>-e</option>/<option>--env-check</option></title>
+       <para>
+        Check if hostname is resolvable, time service, watchdog, and
+       firewall.
+       </para>
+      </formalpara>
+     </listitem>
+     <listitem>
+      <formalpara>
+       <title>Cluster State Check <option>-c</option>/<option>--cluster-check</option></title>
+       <para>
+        Check cluster service, &stonith;/fence configuration, nodes,
```

Comment:
Same here: Please elaborate a bit more.

Reply:
@liangxin1300 I've tried to read the source code, but I guess, it's easier to ask you directly. :wink: 

Could you elaborate a bit?

Reply:
@tomschr @taroth21 
More descriptions can be:
* Check cluster service
   Check whether cluster service(pacemaker/corosync) is enabled and running;
* Check stonith
  Check whether stonith is enabled;
  Check whether stonith related resources is configured and started;
  If we configured sbd, check whether sbd service is started
* Check nodes
  Show current DC node
  Check whether cluster have quorum
  Show Online/OFFLINE/UNCLEAN nodes if cluster has
* Check resources
  Show number of Started/Stopped/FAILED resources
```

Reply:
And for environment check, that is, `-e/--env-check`, more details can be:
#### Check the local node cluster related environment
* Check hostname resolvable
* Check whether time service is enabled and started
* Check whether this node configured watchdog
* Check whether firewalld service is started and whether cluster related ports are opened

Reply:
Thanks @liangxin1300! Much appreciated!

I've added it as a list of questions.